authority-portal-frontend/src/app/common/components/form-elements/application-roles-tooltip/application-roles-tooltip.component.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2024 sovity GmbH
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Contributors:
+ *      sovity GmbH - initial implementation
+ */
+import {Component} from '@angular/core';
+
+@Component({
+  selector: 'app-application-roles-tooltip',
+  templateUrl: './application-roles-tooltip.component.html',
+})
+export class ApplicationRolesTooltipComponent {
+  // may need refinements
+  applicationRolesTooltip = `
+AUTHORITY USER
+has the rights to see the list and details of all organizations, invite new organizations, accept & reject pending requests
```

Comment:
"Can view all participant details, invite new participants and manage pending registration requests"

Reply:
done